### PR TITLE
Issue#28 fix contributing file

### PR DESCRIPTION
**Pull request by romulogoncalves**

### CODE REVIEW

File: {{cookiecutter.project_slug}}/CONTRIBUTING.rst [hunk 1]
```diff
@@ -279,24 +279,7 @@ Tips
...
-
-$ bump2version patch # possible: major / minor / patch
-$ git push
-$ git push --tags
-
-Travis will then deploy to PyPI if tests pass.
+$ pytest tests.test_{{ cookiecutter.project_slug }} -k <test_name_prefix>
 
 Code of Conduct
 ---------------
```
Comment (DaRa42) [inline {{cookiecutter.project_slug}}/CONTRIBUTING.rst:282]: we have a seperate code of conduct file now, should we remove that part?

Comment (romulogoncalves) [reply {{cookiecutter.project_slug}}/CONTRIBUTING.rst:282]: Really, where is it?

Comment (DaRa42) [reply {{cookiecutter.project_slug}}/CONTRIBUTING.rst:282]: {{cookiecutter.project_slug}}/CODE_OF_CONDUCT.rst

Comment (romulogoncalves) [reply {{cookiecutter.project_slug}}/CONTRIBUTING.rst:282]: Yes, I know and it is pointing to it. Check line 287.

Comment (DaRa42) [reply {{cookiecutter.project_slug}}/CONTRIBUTING.rst:282]: oh, right, sorry

**Review by DaRa42 — APPROVED** [review]

**Pull request merged**